eng/targets/Npm.Common.targets

```diff
@@ -22,7 +22,7 @@
 
   <ItemGroup>
     <TSFiles Include="src\**\*.ts" />
-    <TSFiles Include="test\**\*.ts" />
+    <TSFiles Include="tests\**\*.ts" />
```

Comment:
Don't we need both folders due to the inconsistencies in this repo?

Reply:
Not currently. All of our npmproj test projects are plural